src/blogtoblog/BlogToBlogImporter.ts

```diff
@@ -113,7 +113,7 @@ export default class BlogToBlogImporter extends PageImporter {
       dateTitle.textContent = 'Publication Date';
       dateRow.append(dateTitle);
       const dateData = document.createElement('td');
-      dateData.textContent = date;
+      dateData.textContent = date.trim();
```

Comment:
Nitpicking: why adding the `trim()` here and not when setting the `date` value ? There is already a `replace()` there...